iio.h

```diff
@@ -84,6 +84,12 @@ struct iio_context_info;
 struct iio_scan_context;
 struct iio_scan_block;
 
+#define	IIO_CHAN_TYPE_UNKNOWN INT_MAX
+
+#if __has_include(<linux/iio/types.h>)
```

Comment:
Hmm, according to [this](https://gcc.gnu.org/onlinedocs/cpp/_005f_005fhas_005finclude.html), the right way of doing this is to first test for the existence of the macro... I'm not sure this will actually build when the compiler does not support `__has_include`